core/src/main/java/discord4j/core/object/entity/Webhook.java

```diff
@@ -142,7 +187,7 @@ public Snowflake getCreatorId() {
      * by, if present. If an error is received, it is emitted through the {@code Mono}.
      */
     public Mono<User> getCreator() {
```

Comment:
Is the `user` in `WebhookData` good enough to construct a `User` here? If so, we could probably just get rid of `getCreatorId()` and make this return `User` directly.

Reply:
Yes, it is!